.circleci/config.yml

```diff
@@ -369,7 +369,7 @@ jobs:
       - build_setup
       - restore_cargo_package_cache
       - run:
-          command: RUST_BACKTRACE=1 cargo xcheck -j 16 --production
+          command: RUST_BACKTRACE=1 cargo xcheck -j 16 --subset production
```

Comment:
perhaps --members so it's clear what this is a subset of?   Will make more sense if you read the comments below this.

Reply:
Yeah I like `--members`.

Reply:
Done.